CITATION.cff

```diff
@@ -0,0 +1,71 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Evita Module Template
+message: >-
+  If you use this module, please cite it using the metadata
+  from this file.
+type: software
+authors:
+  - given-names: Jane
+    family-names: Doe
+    email: jane.doe@example.com
+    affiliation: Acme University
+    orcid: 'https://orcid.org/1234-5678-0910-1112'
+  - given-names: John
+    family-names: Doe
+    email: john.doe@example.com
+    affiliation: Foo Institute
+    orcid: 'https://orcid.org/1211-1009-8765-4321'
+identifiers:
+  - type: url
+    value: >-
+      https://www.hpc-certification.org/wiki/skill-tree/bda/2/6/b
+    description: BDA2.6 Dask
+  - type: url
+    value: >-
+      https://www.hpc-certification.org/wiki/skill-tree/sd/1/1/b
+    description: SD1.1 Programming Languages
+  - type: url
+    value: >-
+      https://www.hpc-certification.org/wiki/skill-tree/sd/1/3/b
+    description: SD1.3 Efficient Algorithms and Data Structures
```

Comment:
This is how I linked to skill-tree for now, by using "identifiers". It does not have to be a url, but it can also mean a short code